akka-stream/src/main/resources/reference.conf

```diff
@@ -19,6 +19,14 @@ akka {
       # or full dispatcher configuration to be used by ActorMaterializer when creating Actors.
       dispatcher = "akka.actor.default-dispatcher"
 
+      # FQCN of the MailboxType. The Class of the FQCN must have a public
+      # constructor with
+      # (akka.actor.ActorSystem.Settings, com.typesafe.config.Config) parameters.
+      # defaults to the single consumber mailbox for better performance.
+      default-mailbox {
```

Comment:
Why is it named `default-mailbox`? Can there be more than one type for Akka Streams? Would it make more sense to just name it `mailbox`?

Reply:
In the akka typed it's defaultMailbox, so...

Reply:
yes, but there it means "default" if not defined in another way for a specific actor. Here we don't have a way to define it in other ways than the global config, if I understand correctly. We could leave it as `default-mailbox` if we think it would be likely that we add some stream attribute for this -- but Johan already voted against that.

Also, there is a `dispatcher` config property, not `default-dispatcher`.